Loop54.Shared/Model/Request/GetEntitiesByAttributeRequest.cs

```diff
@@ -11,40 +14,71 @@ public class GetEntitiesByAttributeRequest : Request
     {
         public GetEntitiesByAttributeRequest(string attributeName, string attributeValue)
         {
-            Attribute.Name = attributeName;
-            Attribute.Value = attributeValue;
+            Attribute = new AttributeNameValuePairSingle()
+            {
+                Name = attributeName,
+                Value = attributeValue
+            };
         }
-        
+
+        public GetEntitiesByAttributeRequest(string attributeName, string[] attributeValue)
+        {
+            Attribute = new AttributeNameValuePairMultiple()
+            {
+                Name = attributeName,
+                Value = attributeValue
+            };
+        }
+
         /// <summary>
         /// The attribute name-value-pair to find entities connected to. Note: this attribute needs 
         /// to be indexed in the engine. See the endpoint /getIndexedAttributes.
         /// </summary>
-        public AttributeNameValuePair Attribute { get; set; } = new AttributeNameValuePair();
+        public AttributeNameValuePair Attribute { get; set; }
```

Comment:
Now I get what you mean, yes it would be breaking if the consumers does any processing on `Attribute.Value`. But I think that is fine since it would be an compilation error.

Reply:
Hmm, I actually didn't think about that. I don't think any customers to, but just to be safe we should probably call this 6.0.0 instead.